addon/components/plaid-line/component.js

```diff
@@ -44,8 +47,8 @@ const PlaidLineComponent = Ember.Component.extend(GroupElement, {
   curve: curveMonotoneX,
 
   didRender() {
-    const pathAttrs = this.getProperties(w('stroke strokeWidth strokeOpacity fill fillOpacity'));
-    const line = this.selection.select('path.line');
```

Comment:
Personally I prefer the use of `const` unless the variable will be changed. It's a tighter contract with the developer. Do you have any alternate reasons not to use it?